apps/frontend/public/static/export/template.html

```diff
@@ -1,271 +1,637 @@
+                  {{/controlTags}}
+                </div>
+                <!-- Control ID -->
+                <span
+                  class="hidden w-32 grid-cols-1 gap-y-2 break-all px-6 sm:grid lg:w-36"
+                >
+                  {{hdf.wraps.id}}
+                </span>
+                <!-- Control severity indicator -->
+                <div
+                  class="hidden w-28 grid-flow-col items-center rounded border border-gray-300 bg-gray-100 px-2 py-1 text-black lg:grid"
+                >
+                  <span>{{{controlSeverity.icon}}}</span>
+                  <span>{{controlSeverity.severity}}</span>
                 </div>
+                <!-- Control title -->
+                <span class="w-1/2 break-all px-6 sm:break-words"
+                  >{{hdf.wraps.title}}</span
+                >
+                <!-- Control tags -->
+                <div
+                  class="hidden grid-cols-1 gap-x-6 gap-y-2 lg:grid xl:grid-cols-2"
+                >
+                  {{#controlTags}}
+                  <div class="w-24">
+                    <span
+                      class="rounded border border-gray-300 bg-gray-100 px-2 py-1 text-black"
+                      >{{.}}</span
+                    >
+                  </div>
+                  {{/controlTags}}
+                </div>
+
+                <span
+                  class="-mr-1 ml-auto h-5 w-5 shrink-0 rotate-[-180deg] fill-[#336dec] transition-transform duration-200 ease-in-out group-[[data-te-collapse-collapsed]]:mr-0 group-[[data-te-collapse-collapsed]]:rotate-0 group-[[data-te-collapse-collapsed]]:fill-[#212529] motion-reduce:transition-none"
+                >
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke-width="1.5"
+                    stroke="currentColor"
+                    class="h-6 w-6"
+                    role="img"
+                    aria-hidden="true"
+                  >
+                    <path
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      d="M19.5 8.25l-7.5 7.5-7.5-7.5"
+                    />
+                  </svg>
+                </span>
+              </button>
+            </div>
+
+            <!-- Accordion body def -->
+            <div
+              class="multi-collapse-inner px-5 py-4"
+              id="control-accordion-body-{{controlID}}"
+              data-te-collapse-item
+              data-te-collapse-show
+              aria-labelledby="control-accordion-header-{{controlID}}"
+            >
+              <!-- Control description -->
+              {{data.desc}}
+              <hr class="my-3" />
+
+              <!-- Control tables -->
+              <div>
+                <!-- Tests -->
+                <h5 class="text-lg font-bold">Test Results</h5>
+                <table
```

Comment:
usually you wanna move away from using tables for layout.  i think the situation we're in is currently like a 50/50 sorta situation where i can see why you might wanna use it, but could also see why we shouldn't.

anyways, make sure that the styling for the columns is consistent:
<img width="276" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/25f7b57b-95a1-4292-b930-b1c1a9548e9c">

here you can see how the left side is centered vertically whereas the right side is bottom aligned

i think a reason to move away from tables is we could make it more vertically oriented so that it'd look like the following:
<img width="577" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/4fcc3d00-63a7-40e3-b8b5-8bab06d9cb2c">

cause look at how much horizontal space is wasted in the mobile view port just to have that left column

<img width="364" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/a5838015-fd97-4d6a-ad84-f224603f35e2">

i'm not opposed to the zebra style shading still being a thing tho

Reply:
Let's get other folks' perspective on this.  @ChrisHinchey @wdower do yall have opinions?

Let's bring this up at standup as well (or ask around in the lab), but if Chris or Will have opinions that we like before we bring it up to the big group, we can probs save some time.

Reply:
Changed as requested.